docs/features.md

```diff
@@ -0,0 +1,88 @@
+# Features
+
+This document lists and briefly explains the functionality offered by
+Apisonator.
+
+- [Authorization of applications](#authorization-of-applications)
+- [Rate-limit](#rate-limit-of-applications)
+- [Reporting of usage stats](#aggregation-of-usage-stats)
+- [Reporting of response codes](#aggregation-of-response-codes)
+- [Alerts](#alerts)
+- [Events](#events)
+- [Real-time traffic](#real-time-traffic)
+- [Integration errors](#integration-errors)
+- [Usage metrics of the Porta master account](#usage-metrics-of-the-porta-master-account)
+- [Internal management API](#internal-management-api)
+
+## Authorization of applications
+
+Requests are authorized based on some credentials that identify a service
+(provider key, service token) and some credentials that identify an application
+(app ID + app key, user key, oauth token).
+
+## Rate-limit of applications
```

Comment:
Comment that it's not just a global rate-limit, but per metric... and multiple may apply.

Calculation of remaining quota per metric also (which is complex and led to some troubles also?)